react/features/base/jwt/functions.ts

```diff
@@ -165,15 +197,28 @@ export function validateJwt(jwt: string) {
                         && featureValue !== 'true'
                         && featureValue !== 'false'
                     ) {
-                        errors.push(`- Invalid value for feature: ${feature}`);
+                        errors = {
+                            ...errors,
+                            [`- Invalid value for feature: ${feature}`]:
+                                `${i18n.t('dialog.tokenAuthFailedReason.featureValueInvalid')}: ${feature}.`
+                        };
                     }
                 } else {
-                    errors.push(`- Invalid feature: ${feature}`);
+                    errors = {
+                        ...errors,
+                        [`- Invalid feature: ${feature}`]:
+                            `${i18n.t('dialog.tokenAuthFailedReason.featureInvalid')}: ${feature}.`
+                    };
                 }
             });
         }
     } catch (e: any) {
-        errors.push(e ? e.message : '- unspecified jwt error');
+        const message = e?.message;
+
+        errors = {
+            ...errors,
+            [message || '- unspecified jwt error']: message || i18n.t('dialog.tokenAuthFailedReason.unspecifiedError')
```

Comment:
I wouldn't pass the error message here. The catch-all is there mostly as a safely net. If it trips it's probably a bug in our parsing code I think.